recipes/prosic/1.0/meta.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: prosic
+  version: "1.0"
+
+build:
+  number: 1
+  string: "py{{CONDA_PY}}_gsl{{CONDA_GSL}}_gmp{{CONDA_GMP}}_{{PKG_BUILDNUM}}"
+  skip: True # [not py3k]
+
+source:
+  git_url: https://github.com/PROSIC/PROSIC.git
```

Comment:
Indeed, this is the copy of the old recipe, hence I did not see it. I'll fix it.